examples/multi-sheet-example/sheets/organizations.ts

```diff
@@ -0,0 +1,81 @@
+// example Sheet with how you might import custom and reuasable fields and recordCompute functions.
+
+import { NumberField, Sheet, TextField } from '@flatfile/configure'
+
+import { FlatfileRecord, FlatfileRecords } from '@flatfile/hooks'
+
+// Examples of custom defined fields
+import customUrlField from '../fields/url'
+import customCountryField from '../fields/country'
+
+// Examples of custom defined data hooks
+import joinFieldsRecordCompute from '../data-hooks/join-fields-record-compute'
+import splitFieldRecordCompute from '../data-hooks/split-field-record-compute'
```

Comment:
@alexhollenbeck @paddymul  -- when I originally made the simple data-hook examples, I don't think we had the built in ability to split name fields :)

Not sure if it still makes sense to include these?

Reply:
Include them for now in the name of expediency.  The NameField (built on top of composite fields) replaces this need.  Most users will reach for some code like this though